source/config/__init__.py

```diff
@@ -738,7 +742,7 @@ def renameProfile(self, oldName, newName):
 		if oldName.lower() != newName.lower() and os.path.isfile(newFn):
 			raise ValueError("A profile with the same name already exists: %s" % newName)
 
-		os.rename(oldFn, newFn)
+		osreplace.replace(oldFn, newFn)

```

Comment:
Hmm, after looking at the code again, the "same name" case is clearly handled here. SO, I will use os.rename instead.